pkg/metrics/exporter.go

```diff
@@ -21,8 +22,46 @@ var (
 
 const prometheusExporter = "prometheus"
 
-func NewMetricsExporter() error {
-	ce := getCurMetricsExporter()
+type Manager struct {
```

Comment:
Manager is going to get confusing. Maybe Runner or some other name?

Reply:
changed name to `Runner`